catalog/spatial/wps/spatial-wps-endpoint/src/test/java/org/codice/ddf/spatial/ogc/wps/process/endpoint/WpsEndpointTest.java

```diff
@@ -0,0 +1,572 @@
+    ct.setValue("test");
+    executeRequestType.setIdentifier(ct);
+    executeRequestType.setResponse("document");
+    executeRequestType.setMode("sync");
+    DataInputType dataInputType = new DataInputType();
+    Data data = new Data();
+    data.setContent(Collections.singletonList("argument"));
+    data.setEncoding("UTF-8");
+    data.setMimeType("text/plain");
+    data.setSchema("test-schema.txt");
+    dataInputType.setData(data);
+    dataInputType.setId(literalDataDesc.getId());
+    executeRequestType.setInput(Collections.singletonList(dataInputType));
+    OutputDefinitionType dataOutputType = new OutputDefinitionType();
+    dataOutputType.setEncoding("UTF-8");
+    dataOutputType.setId(complexDataDesc.getId());
+    dataOutputType.setMimeType("text/plain");
+    dataOutputType.setSchema("test-schema.txt");
+    dataOutputType.setTransmission(DataTransmissionModeType.REFERENCE);
+    executeRequestType.setOutput(Collections.singletonList(dataOutputType));
+
+    Response response = wpsEndpoint.execute(executeRequestType);
+    verify(simpleProcess).syncExecute(any(ExecutionRequest.class));
+    assertThat(response.getEntity(), is(not(nullValue())));
+  }
+
+  @Test
+  public void getStatus() throws Exception {
+    GetStatus getStatus = new GetStatus();
+    getStatus.setJobID("1234");
+    StatusInfo statusInfo = wpsEndpoint.getStatus(getStatus);
+    assertThat(statusInfo, is(not(nullValue())));
+  }
+
+  @Test
+  public void getResult() throws Exception {
+    GetResult getresult = new GetResult();
+    getresult.setJobID("1234");
+    Result result = wpsEndpoint.getResult(getresult);
+    assertThat(result, is(not(nullValue())));
+  }
+
+  @Test
+  public void dismiss() throws Exception {
+    Dismiss dismiss = new Dismiss();
+    dismiss.setJobID("1234");
+    StatusInfo statusInfo = wpsEndpoint.dismiss(dismiss);
+    assertThat(statusInfo, is(not(nullValue())));
+  }
+
+  @Test
+  public void asyncExecuteMissingArgs() throws Exception {
+    thrown.expect(WpsException.class);
+    thrown.expectMessage("Too few input items have been specified.");
+    ExecuteRequestType executeRequestType = new ExecuteRequestType();
+    CodeType ct = new CodeType();
+    ct.setValue("test");
+    executeRequestType.setIdentifier(ct);
+    executeRequestType.setResponse("document");
+    executeRequestType.setMode("async");
+    executeRequestType.setInput(Collections.emptyList());
+    OutputDefinitionType dataOutputType = new OutputDefinitionType();
+    dataOutputType.setId(complexDataDesc.getId());
+    dataOutputType.setEncoding("UTF-8");
+    dataOutputType.setMimeType("text/plain");
+    dataOutputType.setSchema("test-schema.txt");
+    dataOutputType.setTransmission(DataTransmissionModeType.REFERENCE);
+    executeRequestType.setOutput(Collections.singletonList(dataOutputType));
+
+    Response response = wpsEndpoint.execute(executeRequestType);
+    verify(simpleProcess).asyncExecute(any(ExecutionRequest.class));
```

Comment:
Don't need these last two lines since you're just checking that `execute()` throws an exception.